theme.qml

```diff
@@ -223,6 +231,14 @@ FocusScope {
             menu_games:"games"
         },
         "de": {
+            global_dimm: "dimm",
+            global_bright: "bright",
+            global_Dark: "Dark",
+            global_Light: "Light",
+            global_OzoneDark: "OzoneDark",
+            global_SteamOS: "SteamOS",
+            global_mono: "mono",
+            global_color: "color",

```

Comment:
```suggestion
            settings_collection_accentColor_dimm: "Leuchtend",
            settings_collection_accentColor_bright: "Gedämpft",
            settings_general_colorScheme_Dark: "Hell",
            settings_general_colorScheme_Light: "Dunkel",
            settings_general_colorScheme_OzoneDark: "OzoneDark",
            settings_general_colorScheme_SteamOS: "SteamOS",
            settings_general_logoVariant_mono: "Einfarbig",
            settings_general_logoVariant_color: "Bunt",
```
global is for text strings which can appear in many places in the theme. I assume these strings will never be needed outside the settings screen. Therefore I would have adapted the naming scheme to this one

Reply:
ok good information